lib/hashie/extensions/persistable.rb

```diff
@@ -0,0 +1,38 @@
+  module Hashie
+    module Extensions
+      module Persistable
+        CANNOT_INCLUDE = 'Peristable can only be used on classes with Hashie::Extensions::HashInitializer'
```

Comment:
Agreed. As long as we do it the same way everywhere. We can leave nice(r) errors for some future implementation.